src/Query.php

```diff
@@ -571,17 +554,9 @@ public function where($field, $cond = null, $value = null, $kind = 'where', $num
         }
 
         // Array as first argument means we have to replace it with orExpr()
-        if ($num_args === 1 && is_array($field)) {
-            // or conditions
-            $or = $this->orExpr();
-            foreach ($field as $row) {
-                if (is_array($row)) {
-                    $or->where(...$row);
-                } else {
-                    $or->where($row);
-                }
-            }
-            $field = $or;
+        // remove in v2.5
+        if (is_array($field)) {
+            throw new Exception('Array input / OR conditions is no longer supported');
```

Comment:
This was easy and handy method how to define OR conditions :(

Reply:
and how to possiblyand silently pass mistakes